app/build.gradle.kts

```diff
@@ -115,6 +115,7 @@ dependencies {
 
     // Coil
     implementation(libs.coil.compose)
+    implementation(libs.richeditor.compose)
```

Comment:
What is the use of the dependency in this PR?
Please remove it @Ismail-AD

Reply:
@aritra-tech it is done  do review and merge